README.md

```diff
@@ -10,8 +10,7 @@ A PostgreSQL OAuth 2.0 Server.
 
 **starting server:**
 
-```sh
-./bin/oauth2-server-pg server --help
+```./bin/oauth2-server-pg.js start --help
```

Comment:
Whoops, this doesn't quite render properly ([view on your branch](https://github.com/bitdivine/oauth2-server-pg#usage)). I think we need to keep the newline and language identifier:

    ```sh
    ./bin/oauth2-server-pg.js start --help
    ```